lib/plugins/chatops.rb

```diff
@@ -21,6 +21,14 @@ def notify(message)
                            message,
                            http_options: @args[:http_options] || {},
                            icon_emoji: @args[:icon_emoji]).notify
+      when 'rocketchat'
+        LOGGER.info("Sending Rocket.Chat webhook message to #{@url}")
+        Chatops::Rocketchat.new(@channel,
```

Comment:
Personally I prefer it when all params in new() are in the next line, limits the indentation to one level

Reply:
I believe that is against the Ruby Style Guide and I don't think we turned off those cops in Rubocop. I saw failures related to trying to put parameters in-between open/closing parentheses on their own lines. It wants the first and last parameters on the same line as the open/close.

Same goes when calling a class method after the `.new()`, it wants the final parameter, closing parenthesis, and method call all on the same line.

Reply:
I found a solution that should satisfy you, @bastelfreak. I'm pretty sure it's close to what you are asking.